mama_ng_control/apps/vumimessages/tests.py

```diff
@@ -319,8 +323,16 @@ def test_delete_inbound_data(self):
         d = Inbound.objects.filter(id=existing).count()
         self.assertEqual(d, 0)
 
+    @responses.activate
     def test_event_ack(self):
         existing = self.make_outbound()
+
+        # Setup response from scheduler
+        responses.add(
+            responses.DELETE,
+            "http://127.0.0.1:8000/mama-ng-scheduler/rest/messages/1",
+            json.dumps({}), status=200, content_type='application/json')
+
```

Comment:
Do you want to add an assert here somewhere to check that the subscription metadata for scheduler_message_id has been set to ""?